tests/test_registrations/test_views.py

```diff
@@ -0,0 +1,476 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+'''Views tests for the OSF.'''
+
+from __future__ import absolute_import
+import datetime as dt
+import mock
+import httplib as http
+from dateutil.parser import parse as parse_date
+
+from nose.tools import *  # noqa PEP8 asserts
+
+from modularodm import Q
+
+from framework.exceptions import HTTPError
+from framework.auth import Auth
+
+from website.models import Node, MetaSchema, DraftRegistration
+from website.project.metadata.schemas import ACTIVE_META_SCHEMAS, _name_to_id
+from website.util import permissions, api_url_for
+from website.project.views import drafts as draft_views
+
+from tests.factories import (
+    NodeFactory, AuthUserFactory, DraftRegistrationFactory, RegistrationFactory
+)
+from tests.test_registrations.base import RegistrationsTestBase
+
+from tests.base import get_default_metaschema
+DEFAULT_METASCHEMA = get_default_metaschema()
+
+class TestRegistrationViews(RegistrationsTestBase):
+
+    def test_node_register_page_not_registration_redirects(self):
+        url = self.node.web_url_for('node_register_page')
+        res = self.app.get(url, auth=self.user.auth)
+        assert_equal(res.status_code, http.FOUND)
+
+    @mock.patch('website.archiver.tasks.archive')
```

Comment:
Your patching of unnecessary tasks is appreciated!

But just in case some tests (like the above, L33) don't do what is expected: have you considered patching the archiver tasks there as well? (just in case they would otherwise have run, unintentionally)